rich/_lru_cache.py

```diff
@@ -1,12 +1,16 @@
-from collections import OrderedDict
-from typing import Dict, Generic, TypeVar
-
+from typing import Dict, Generic, TypeVar, TYPE_CHECKING
+import sys
 
 CacheKey = TypeVar("CacheKey")
 CacheValue = TypeVar("CacheValue")
 
+if sys.version_info < (3, 8):
```

Comment:
```suggestion
if sys.version_info < (3, 9):
```

Oops, applied both changes, didn't mean to add this one.